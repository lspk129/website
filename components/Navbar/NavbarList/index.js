import styled from 'styled-components';

const NavbarList = styled.div`
  text-align: center;
  position: fixed;
  grid-row: 2 / 3;
`;

export default NavbarList;
