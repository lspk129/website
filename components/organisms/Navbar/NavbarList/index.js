import styled from 'styled-components';

const NavbarList = styled.div`
  display: flex;
  align-items: center;
  text-align: center;
`;

export default NavbarList;
