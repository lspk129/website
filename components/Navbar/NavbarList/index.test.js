import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import NavbarList from './';

describe('NavbarList', () => {
  const shallowWrapper = shallow(<NavbarList />);

  it('renders correctly', () => {
    const wrapper = renderer.create(<NavbarList />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders div element', () => {
    expect(shallowWrapper).toBePresent();
    expect(shallowWrapper).toHaveTagName('div');
  });
});
