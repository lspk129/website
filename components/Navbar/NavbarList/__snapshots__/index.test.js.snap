// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarList renders correctly 1`] = `
<div
  className="sc-bdVaJa igOxDQ"
/>
`;
